### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by fuzzyTew** [kvasir-io/mpl#44]
It would be great to have a quick list somewhere of the entire API, with a couple words for each item.

**Comment by odinthenerd** [comment]
Yes we are working on it: http://kvasir.io/mpl/doc/standardese/standardese_entities.html
